product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/utils/XMLComparatorInterop.java

```diff
@@ -0,0 +1,212 @@
+package org.wso2.carbon.esb.scenario.test.common.utils;
+
+import org.apache.axiom.om.OMAttribute;
+import org.apache.axiom.om.OMElement;
+import org.apache.axiom.om.OMNamespace;
+import org.apache.axiom.om.OMNode;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.util.Iterator;
+import java.util.Vector;
+
+/**
+ * Util class for comparing two OMElements
+ */
+public class XMLComparatorInterop {
```

Comment:
What is the meaning with "Interop"?

Reply:
Copied from Axis2. 
https://alvinalexander.com/java/jwarehouse/axis2-1.3/modules/integration/manual/src/test/interop/util/XMLComparatorInterop.java.shtml

Will rename to XMLComparator